tf_agents/agents/ppo/ppo_agent.py

```diff
@@ -164,6 +165,9 @@ def __init__(self,
       reward_norm_clipping: Value above an below to clip normalized reward.
       normalize_observations: If true, keeps moving mean and variance of
         observations and normalizes incoming observations.
+      train_on_partial_episodes: If True (default False), then data from
```

Comment:
i wonder if we can just always train on partial episodes.  @oars wdyt?  can we get rid of this arg and always assume true?

Reply:
I think it should be fine. Previous use cases will still work as they are feeding full episodes already.